### DISCUSSION TITLE
I Cant Use Classes (d-flex)

### DISCUSSION THREAD

**Discussion by cengizilhan** [reactstrap/reactstrap#2514] [Q&A]
i installed bootstrap & react-bootstrap via npm but i cant use "d-flex" class. what can i do for use Bootstrap classes?

What is best practice?

 "bootstrap": "^5.1.3",
    "react": "^18.1.0",
    "react-bootstrap": "^2.3.1",

**Comment by davidacevedo** [comment]
We don't import bootstrap for you. We assume that you have followed the bootstrap documentation and imported the bootstrap styles in your html file https://getbootstrap.com/. You can also do this by importing through webpack, but it's really up to the developers to get the styles available globally
